src/kernel.jl

```diff
@@ -22,7 +22,7 @@ ENV["COLUMNS"] = get(ENV, "COLUMNS", 80)
 
 IJulia.init(ARGS)
 
-let startupfile = abspath(homedir(), ".julia", "config", "startup_ijulia.jl")
+let startupfile = !isempty(DEPOT_PATH) ? abspath(DEPOT_PATH[1], "config", "startup_ijulia.jl") : ""
```

Comment:
If the depot path is empty, I don’t think we should try to load a startup file at all